### ISSUE TITLE
no-ex-assign going wild

### ISSUE THREAD

**Issue by zelein** [buzinas/tslint-eslint-rules#114]
The `no-ex-assign` is flagging code that isn't even within a catch statement.

It's flagging each chai.expect statement in this basic test.
```
describe("The Basic Config", (): void => {
    it("shall have specific properties attached to it.", (done: (error?: Error) => void): void => {
        chai.expect(config).to.contain.all.keys(["extends", "rules", "rulesDirectory"]);

        chai.expect(config.extends).to.be.an("array");
        chai.expect(config.rulesDirectory).to.be.an("array");
        chai.expect(config.rules).to.be.an("object");

        done();
    });
});
```

It's doing this in many other places where there are no existing catch statements.

**Comment by buzinas** [comment]
It's a known issue and #54 is already open for that. Also, there is a pending PR for fixing it (#89).

**Comment by buzinas** [comment]
The issue was fixed, and the problem won't help anymore in the next release.

**Comment by buzinas** [comment]
@zelein v1.6.0 released
